web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -73,6 +83,7 @@ export const SidebarCardsList = (props: SidebarCardsListProps): ReactElement =>
                 setHiddenAccordionIsOpen((prevAccordionStatus) => {
                   return !prevAccordionStatus;
                 });
+                scrollAccordionToTop(accordionRef.current, { isDesktop: isDesktopAndUp });
```

Comment:
[sand] I think it might make more sense to have this only fire when the menu opens. Right now it's also firing on close.

Reply:
Agree, but I think this is a pebble. I don't think we should merge the code until this is addressed, because right now it scrolls to an empty part of the page on close, which is antithetical to the intentions set out by the user story.